e2e_playwright/st_markdown.py

```diff
@@ -200,3 +200,7 @@ def draw_header_test(join_output):
     """,
     help="This is example tooltip displayed on latex.",
 )
+
+st.markdown(
+    "Images in markdown should keep inside the container width:\n\n![image](./app/static/streamlit-logo.png)"
```

Comment:
nit: "Images in markdown should keep inside the container width" -> "Images in markdown should stay inside the container width"

Reply:
Updated 👍